libft/ft_atoi.c

```diff
@@ -0,0 +1,54 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_atoi.c                                          :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: fcaetano <fernandacunha@id.uff.br>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/05/09 15:39:31 by fcaetano          #+#    #+#             */
+/*   Updated: 2022/06/08 10:08:30 by fcaetano         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+static char	*ft_wspaces(char *str)
+{
+	while (*str && (*str == ' ' || (*str >= '\t' && *str <= '\r')))
+		str++;
+	return (str);
+}
+
+static int	ft_catnbr(char *str)
+{
+	unsigned int	c;
+	unsigned int	mult;
+	int				result;
+
+	c = 0;
+	while (str[c] >= '0' && str[c] <= '9' && str[c] != 0)
+			c++;
```

Comment:
Wrong indentation.

Reply:
Ok, will work on it! (:

Reply:
Done! Also fixed some norminette erros in main.c